operators/merge_from_scene.py

```diff
@@ -0,0 +1,53 @@
+import bpy
+from .utils.doc import doc_name, doc_idname, doc_brief, doc_description
+
+
+class MergeFromSceneStrip(bpy.types.Operator):
+    """
+    Merge into the current scene meta data from a scene strip re-creating strips and markers.
```

Comment:
Copies all the sequences and markers from within the scene strip to the active scene, and removes the target scene. 
⚠ Currently, all animation on sequences will be lost